src/App.js

```diff
@@ -1,5 +1,37 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+import { MESSAGE } from './constants/message.js'
+import {
+	inputCarsNameAndValidate,
+	inputTryCountAndValidate,
+	printResult,
+	racingResult,
+	winner,
+} from './utils/index.js'
+
 class App {
-  async play() {}
+	async play() {
+		//자동차 이름 입력
+		MissionUtils.Console.print(MESSAGE.START)
+		const carsNameSplit = await inputCarsNameAndValidate()
+		let carsInfo = new Map(carsNameSplit.map(carName => [carName, 0]))
```

Comment:
```suggestion
		const carsInfo = new Map(carsNameSplit.map(carName => [carName, 0]))
```
`carsInfo`를 `let` 키워드로 선언하신 이유가 있으신가요??

Reply:
`Map`이라면 `const`로 선언하셔도 괜찮을 것 같습니당. 배열처럼 `Map`도 JS에서 **참조 타입**이기 때문에 `const`로 선언해도 내용 변경이 가능합니다

Reply:
아 그렇군요! const로 변경해도되겠네요! 감사합니당~~